src/inject.js

```diff
@@ -219,19 +219,29 @@ export const injectAndGetClassName = (
 ) /* : string */ => {
     styleDefinitions = flattenDeep(styleDefinitions);
 
-    // Filter out falsy values from the input, to allow for
-    // `css(a, test && c)`
-    const validDefinitions = styleDefinitions.filter((def) => def);
-
+    let classNameBits = [];
+    let definitionBits = [];
+    for (let i = 0; i < styleDefinitions.length; i += 1) {
+      // Filter out falsy values from the input, to allow for
+      // `css(a, test && c)`
+      if (styleDefinitions[i]) {
+        classNameBits.push(styleDefinitions[i]._name);
+        definitionBits.push(styleDefinitions[i]._definition);
+      }
+    }
```

Comment:
Hahaha not at all, I was thinking the same thing. We have eslint but it doesn't have much turned on. :P I think ideally it would follow the KA style eslint at https://github.com/Khan/khan-linter/blob/master/eslintrc but that's ginormous so maybe just adding in a few of the common annoying ones would be good...